Project.toml

```diff
@@ -1,18 +1,17 @@
 name = "CategoricalArrays"
 uuid = "324d7699-5711-5eae-9e2f-1d82baa6b597"
-version = "0.7.4"
+version = "0.7.5"
```

Comment:
Please keep the version as-is for now, we'll likely go to 0.8 for the next release.

Reply:
Done. I changed the version back to 0.7.4.